src/main/java/com/redhat/devtools/intellij/qute/psi/core/command/QuteUpdateConfigurationAction.java

```diff
@@ -0,0 +1,167 @@
+    public void actionPerformed(@NotNull AnActionEvent e) {
+        JsonObject configUpdate = getConfigUpdate(e);
+        if (configUpdate != null && e.getProject() != null) {
+            String section = configUpdate.get("section").getAsString();
+            ConfigurationUpdater updater = updaters.get(section);
+            if (updater == null) {
+                throw new UnsupportedOperationException("Updating '" + section + "' is not supported yet!");
+            }
+            updater.updateConfiguration(e.getProject(), configUpdate);
+        }
+    }
+
+    private @Nullable JsonObject getConfigUpdate(@NotNull AnActionEvent e) {
+        @Nullable Command command = e.getData(CommandExecutor.LSP_COMMAND);
+        if (command == null) {
+            return null;
+        }
+        List<Object> arguments = command.getArguments();
+        if (arguments != null && !arguments.isEmpty()) {
+            Object arg = arguments.get(0);
+            if (arg instanceof JsonObject) {
+                return (JsonObject) arg;
+            }
+        }
+        return null;
+    }
+
+    interface ConfigurationUpdater {
+        void updateConfiguration(Project project,  JsonObject value);
+    }
+
+    private static class InspectionConfigurationUpdater implements ConfigurationUpdater {
+
+        protected final String inspectionId;
+
+        InspectionConfigurationUpdater(String inspectionId) {
+            this.inspectionId = inspectionId;
+        }
+
+        @Override
+        public void updateConfiguration(Project project, JsonObject configUpdate) {
+            JsonElement value = configUpdate.get("value");
+            if (value != null) {
+                addToExclusions(project, value.getAsString());
+            }
+        }
+
+        protected void addToExclusions(Project project, @NotNull String value) {
+            InspectionProfile profile = InspectionProfileManager.getInstance(project).getCurrentProfile();
+            InspectionToolWrapper<?, ?> toolWrapper = profile.getInspectionTool(inspectionId, project);
+            if (toolWrapper != null && toolWrapper.getTool() instanceof AbstractDelegateInspectionWithExclusions) {
+                Key<AbstractDelegateInspectionWithExclusions> key = new Key<>(inspectionId);
+                profile.modifyToolSettings(key, getPsiElement(project), (tool) -> {
+                    tool.excludeList.add(sanitize(project, value));
+                });
+            }
+        }
+
+        /**
+         * returns file value path relative to the given project
+         * @param project the reference project
+         * @param value the file path URI to get a relative path from
+         * @return file value path relative to the given project
+         */
+        private String sanitize(@NotNull Project project, @NotNull String value) {
+            if (value.startsWith("file:/")) {
+                try {
+                    URI projectURI = new File(project.getBasePath()).toURI();
```

Comment:
Try to use `LSPIJUtils.toUri(Project project)`, if it doesn't work for you (I think it will have .idea), we need to fix it